robottelo/cli/factory.py

```diff
@@ -1171,14 +1171,17 @@ def make_usergroup(options=None):
 @cacheable
 def make_usergroup_external(options=None):
     """
-    Usage:
+    Usage::
+
         hammer user-group external create [OPTIONS]
 
-    Options:
+    Options::
+
         --auth-source-id AUTH_SOURCE_ID           ID of linked auth source
         --name NAME                               External user group name
         --user-group, --usergroup USER_GROUP_NAME Name to search by
         --user-group-id, --usergroup-id USER_GROUP_ID
+
     """
```

Comment:
@sghai , these changes are to reduce the `warnings` from `make docs-clean docs`.

I know this should be in a different PR, but I could fix it so it's here.